draft-ietf-mls-protocol.md

```diff
@@ -154,6 +154,8 @@ RFC EDITOR PLEASE DELETE THIS SECTION.
 
 draft-04
 
+- Updating the language to be similar to the Architecture document. (\*)
```

Comment:
This doesn't need a `(\*)`, since it's not a breaking change.

Reply:
Fixed.